docs/appendices.txt

```diff
@@ -5,7 +5,7 @@
 [appendix]
 == API Documentation (JavaDoc)
 
-Refer to the JanusGraph API documentation for a complete documentation of all core APIs exposed by JanusGraph.
+Refer to the JanusGraph https://javadoc.io/doc/org.janusgraph/janusgraph-core[API documentation] for a complete documentation of all core APIs exposed by JanusGraph.
```

Comment:
@mbrukman It definitely would be better to link to the correct version. In the case where it's a snapshot version the page will show a "version does not exist" message, though the user could still then change versions on the same page. Is this okay?

https://javadoc.io/doc/org.janusgraph/janusgraph-core/0.2.0-SNAPSHOT